src/compas_rhino/geometry/mesh.py

```diff
@@ -100,16 +100,19 @@ def to_compas(self, cls=None):
             The equivalent COMPAS mesh.
         """
         cls = cls or Mesh
-        faces = []
-        for face in self.faces:
-            if face[0] == face[-1]:
-                faces.append(face[:-1])
-            elif face[-2] == face[-1]:
-                faces.append(face[:-1])
+        mesh = cls()
+
+        for key, vertex in enumerate(self.geometry.Vertices):
```

Comment:
enumeration and key assignment are not necessary because 0-based continuous integers are the default behaviour

Reply:
true! fixed at https://github.com/compas-dev/compas/pull/829/commits/6f7c98532300e9e39037f667c3466127a9f1b192